packages/deployment/bigdipper/bigdipper.sh

```diff
@@ -5,20 +5,20 @@
 trap 'kill $(jobs -p) 2>/dev/null' EXIT
 
 NETWORK_URL=${1-"https://testnet.agoric.com"}
-ncf=`curl -Ss "$NETWORK_URL/network-config"`
-cn=`echo "$ncf" | jq -r '.chainName'`
+ncf=$(curl -Ss "$NETWORK_URL/network-config")
```

Comment:
Apparently so. How confident are you that this change is correctness preserving? Semantically equivalent? Or semantically preferred for some reason?

Reply:
Fully confident it preserves. One reason it's better is it [allows nesting](https://stackoverflow.com/questions/9449778/what-is-the-benefit-of-using-instead-of-backticks-in-shell-scripts).